src/components/Drawer/Drawer.module.css

```diff
@@ -11,34 +11,29 @@
  */
 .drawer {
   display: flex;
+  flex-direction: column;
+  align-items: flex-end;
   justify-content: flex-end;
   visibility: hidden;
-  width: 100%;
+  width: 28rem;
   height: 100vh;
   position: fixed;
   top: 0;
   right: 0;
   z-index: var(--eds-z-index-bottom);
-  opacity: 0;
-  overflow: auto;
   background-color: transparent;
+}
 
-  /**
-  * Active modal
-  */
-  &.eds-is-active {
-    opacity: 1;
-    visibility: visible;
-    z-index: var(--eds-z-index-400);
-  }
-
-  .drawer--show-backdrop & {
-    background-color: rgba(0, 0, 0, 0.4);
-  }
+.drawer:focus-visible {
+  @mixin focus;
+}
 
-  &:focus-visible {
-    @mixin focus;
-  }
+/**
+* Active modal
```

Comment:
Btw should we update these comments to say "Drawer" instead of "Modal"?

Reply:
Yup will do